src/components/layout/LayoutBase.js

```diff
@@ -37,31 +41,41 @@ export const LayoutBase = ({
   previewImage,
   children,
   viewport = "width=device-width, initial-scale=1",
-}) => (
-  <>
-    <Helmet
-      link={FONTS.filter((font) => font.preload).flatMap((font) => ({
-        rel: "preload",
-        href: font.src[0].url,
-        as: "font",
-      }))}
-      meta={[
-        {
-          name: "viewport",
-          content: viewport,
-        },
-      ]}
-    />
-    <Seo
-      title={title}
-      description={description}
-      previewImage={previewImage}
-      path={pageContext.urlPath}
-    />
-    <SkipToContentEl />
-    <ContentEl>{children}</ContentEl>
-  </>
-);
+}) => {
+  mark(PERFORMANCE_MARKS.layout);
+  React.useEffect(() => {
+    const timing = measure(PERFORMANCE_MARKS.layout);
+    emitMetric("page rendered", {
+      ...timing,
+      page: window.location.pathname,
+    });
+  });
```

Comment:
Only use of metrics at the moment. Figure this is a good, simple data point that gets to interactivity and overall time to load, where it's within our control. There's a ton of HTML, so we can't do too much about overall page size

Reply:
`timing` here is `{ duration, hasHighPrecision }`